static/css/style.css

```diff
@@ -1,193 +1,245 @@
+* {
+  -webkit-tap-highlight-color: rgba(black, 0);
+  -webkit-tap-highlight-color: transparent;
+  box-sizing: border-box;
+  text-size-adjust: 100%;
+  font: inherit;
+  font-size: 100%;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
 }
-.ml.small{
-    font-size: 0.6em;
-}
-.footer-logo{
-    height: 60px;
-    margin: 15px;
+
+body {
+  background: #white;
+}
+.navbar {
+  padding: 15px 30px;
+  margin-bottom: 5px;
+  background: #efefef;
+}
+.brand-logo,
+.brand-logo:hover {
+  font-size: 24px;
+  color: #333333;
+  border: 2px solid #333333;
+  padding: 6px 10px;
+  font-weight: 500;
+  text-decoration: none;
+  display: inline-block;
+  margin-top: 5px;
+}
+.nav-logo {
+  height: 60px;
+}
+.backtohome {
+  float: right;
+  color: #333333;
+}
+.contactus {
+  float: right;
+  color: #333333;
+  padding-right: 5px;
+}
+h1.main-logo {
+  font-size: 6rem;
+  display: block;
+  color: #333333;
+  font-weight: 600;
+  margin-top: 0;
+}
+a.home-button {
+  position: relative;
+  margin: 10px;
+  border: 2px solid #333333;
+  color: #333333;
+  border-radius: 5px;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  text-decoration: none;
+  width: 230px;
+  height: 230px;
+  vertical-align: top;
+  display: inline-flex;
+  justify-content: center;
+  flex-direction: column;
+}
+a.home-button:hover {
+  background: #333333;
+  color: #ffffff;
+}
+a.home-button .glyphicon {
+  position: absolute;
+  top: 30px;
+  width: 100%;
+  font-size: 5rem;
+  display: block;
+  margin-bottom: 10px;
+}
+a.home-button .text {
+  position: absolute;
+  top: 90px;
+  width: 100%;
+  padding: 0 10px;
+  font-size: 2rem;
+  font-weight: 500;
+  display: inline-block;
+}
+a.home-button .ml {
+  margin-top: 7px;
+  padding-top: 7px;
+  border-top: 1px dotted #555555;
+  font-size: 1.7rem;
+  font-weight: bold;
+  text-transform: uppercase;
+  display: inline-block;
+}
+p.home-info {
+  font-size: 1.6rem;
+  font-weight: 500;
+  max-width: 800px;
+  margin: 30px auto 0;
+}
+form {
+  width: 96%;
+  margin: 20px auto;
+  max-width: 600px;
+}
+footer {
+  margin: 40px 0;
+}
+.ml.small {
+  font-size: 0.6em;
+}
+.footer-logo {
+  height: 60px;
+  margin: 15px;
 }
+
 @media screen and (max-width: 640px) {
-    .navbar{
-        padding: 4px 6px;
-    }
-    .brand-logo,.brand-logo:hover{
-        font-size: 15px;
-        padding: 3px 6px;
-        margin-top: 0;
-    }
-    h1.main-logo{
-        font-size: 3.5rem;
-        margin-top:10px;
-    }
-    a.home-button .glyphicon{
-        font-size: 3rem;
-        margin-bottom: 15px;
-    }
-    a.home-button{
-        display: inline-block;
-        padding: 20px 10px;
-        margin: 5px 10px;
-        width: 80%;
-        height: auto;
-    }
-    a.home-button .text{
-        font-size: 2rem;
-    }
-    .nav-logo{
-        height: 30px;
-    }
-    .footer-logo{
-        height: 80px;
-        display: block;
-        margin: 15px auto;
-    }
+  .navbar {
+    padding: 4px 6px;
```

Comment:
How about reducing `min-height` ? Considerable gap between logo and container on mobile devices.